slackbridge/bots.py

```diff
@@ -93,9 +96,19 @@ def _change_presence(self, user_bot):
             elif self.raw_message['presence'] == 'active':
                 user_bot.back()
 
+        def _irc_action(self, channel_name, user_bot):
+            user_bot.post_to_irc(
+                user_bot.describe,
```

Comment:
Where does "describe" come from? I can't find any references to it.

Reply:
http://twistedmatrix.com/documents/current/api/twisted.words.protocols.irc.IRCClient.html#describe

It seems like the most appropriate way to do `/me` actions anyway.